lib/sass.js

```diff
@@ -58,11 +58,14 @@ export async function buildCSS(inputs, { tailwindConfig } = {}) {
         to: output,
         map: {
           inline: false,
-          prev: sassResult.map?.toString(),
+          prev: JSON.stringify(sassResult.sourceMap),
         },
       });
 
-      await writeFile(output, postcssResult.css);
+      await writeFile(
+        output,
+        `${postcssResult.css}\n/*# sourceMappingURL=${inputFile}.map */`
```

Comment:
Adding the sourceMapURL here like this is a bit hacky, but `sass.compile` does not add it, and it is documented that it is up to you to add this comment:

https://sass-lang.com/documentation/js-api/interfaces/Options#sourceMap

![image](https://github.com/hypothesis/frontend-build/assets/2719332/263c0f18-d252-420c-8917-1e29bdfdc4de)

I have even found some issues of people reporting this, and some suggesting a solution like the one above.

I have mixed feelings though.

Reply:
ok!